### PR TITLE
[workspace] update repository README.md

### PR DESCRIPTION
**Pull request by Simek**
# Why & How

Update social profiles links, remove deprecated forums, add few more useful links, add initial sections.

# Test Plan

Preview: https://github.com/expo/expo/tree/%40simek/workspace-update-readme?tab=readme-ov-file#expo

Screenshot 2023-12-30 at 18 35 10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +24,40 @@
...
+    <img alt="Expo on LinkedIn" src="https://img.shields.io/badge/Bluesky-1DA1F2?style=for-the-badge&logo=bluesky&logoColor=white" target="_blank" />
+  </a>&nbsp;
+  <a aria-label="Follow @expo on LinkedIn" href="https://www.linkedin.com/company/expo-dev" target="_blank">
+    <img alt="Expo on LinkedIn" src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" target="_blank" />
   </a>
 </p>
-  
----
+
+## Summary
```
Comment (amandeepmittal) [inline README.md:54]: Since this is the first section, let's call it introduction.

```suggestion
## Introduction
```

**Review by amandeepmittal — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -27,20 +24,40 @@
...
+  <a aria-label="Follow @expo on LinkedIn" href="https://www.linkedin.com/company/expo-dev" target="_blank">
+    <img alt="Expo on LinkedIn" src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" target="_blank" />
   </a>
 </p>
-  
----
+
+## Introduction
+
+Expo is an open-source platform for making universal native apps that run on Android, iOS, and the web. It includes a universal runtime and libraries that let you build native apps by writing React and JavaScript. This repository is where the Expo client software is developed, and includes the client apps, modules, apps, CLI, and more. [Expo Application Services (EAS)](https://expo.dev/eas) is a platform of hosted services that are deeply integrated with Expo open source tools. EAS helps you build, ship, and iterate on your app as an individual or a team.
```
Comment (brentvatne) [inline README.md:56]: i'd change this "This repository" sentence to:

> This repository includes the Expo SDK, Modules API, Go app, CLI, Router, documentation, and various other supporting tools.

File: README.md [hunk 3]
```diff
@@ -27,20 +24,40 @@
...
+
+[Click here to view the Expo Community Guidelines](https://expo.dev/guidelines). Thank you for helping keep the Expo community open and welcoming!
```
Comment (brentvatne) [inline README.md:58]: Click here to view the Expo Community Guidelines -> Read the Expo Community Guidelines.

File: README.md (hunk 2 above)
Comment (brentvatne) [reply README.md:56]: ah, i saw this wasn't even written in this pr, so i made the change myself here https://github.com/expo/expo/commit/58250e142307873fca65eb2fa76229fcd17187d4

**Pull request merged**